astroplan/plots/plots.py

```diff
@@ -0,0 +1,294 @@
+        time = time + np.linspace(-12, 12, 100)*u.hour
+
+    # Calculate parallactic angle.
+    declination = target.dec
+    latitude = observer.location.latitude
+    azimuth = observer.altaz(time, target).az
+    hour_angle = target.ra
+
+    numerator = np.sin(azimuth) * np.cos(latitude)
+    denominator = np.cos(declination) * (np.cos(azimuth)*np.cos(hour_angle) + np.sin(azimuth)*np.sin(hour_angle)*np.sin(latitude))
+    parallactic_angle = np.arcsin(numerator/denominator)
+
+    # Some checks & info for labels.
+    assert len(time) == len(parallactic_angle)
+
+    if hasattr(observer, 'name') is False:
+        observer_name = 'No Observer/Location Name'
+    else:
+        observer_name = observer.name
+
+    if hasattr(target, 'name') is False:
+        target_name = str('%.3f' % (random.random()))
+    else:
+        target_name = target.name
+
+    observe_date = time[0].datetime.strftime('%Y-%m-%d')
+    observe_timezone = 'UTC'
+
+    # Plot data.
+    ax.plot_date(
+        time.plot_date,
+        parallactic_angle,
+        linestyle=style_kwargs['linestyle'],
+        color=style_kwargs['color'],
+        label=target_name,
+        )
+    ax.figure.autofmt_xdate()
+
+    # Set labels, title, legend, etc.
+    ax.set_ylabel("Parallactic Angle")
+    ax.set_xlabel("Hour - "+observe_timezone)
+    ax.set_title("Parallactic Angle vs Time | "+observer_name+" | "+observe_date+" "+observe_timezone)
+    ax.legend(shadow=True)
+
+    return ax
+
+
+def plot_sky(target, observer, time, ax=None, style_kwargs=None):
+    """
+    Returns an ax object with a sky plot.
+
+    If an ax object is passed in, plots an additional sky plot on top.
+    Otherwise, creates a new ax object with a sky plot.
+
+    Parameters
+    ----------
+    target: `~astroplan.FixedTarget`
+        The celestial body of interest.
+
+    observer: `~astroplan.Observer`
+        The person, telescope, observatory, etc. doing the observing.
+
+    time : `~astropy.time.Time`
+        An collection of times to use in generating data to plot.
+        Can be scalar (len(time)=1).
+
+    ax : `~matplotlib.axes.Axes` or None
+        Optional.
+
+    style_kwargs : WHAT TYPE IS DICTIONARY? or None
+        Optional.
+        A dictionary with plotting style options.
+    """
+    #######################################################
+    # Pull in input from file & other bits.
+    # Replace this later with input from astroplan objects.
+    date = '2015-01-15'
+    #######################################################
+
+    # Set up axes & plot.
+    if ax is None:
+        # axisbg option sets background color within outer circle (plot bounds).
+        # May need to set a color outside of plot bounds.
+        ax = plt.gca(projection='polar', axisbg='0.96')
+    if style_kwargs is None:
+        # Need to add more style options for ax's with existing plots?
+        style_kwargs = {'marker': 'o', 'color': 'b'}
+
+    # Set up figure.
+    dimensions = (9, 9)
```

Comment:
You can't do `plt.figure` here because that creates a figure and that defeats the whole point of passing in `ax` above.